src/EloquentRepository.php

```diff
@@ -394,6 +394,19 @@ protected function fill(Model $instance): bool
                             'value'    => $value,
                         ];
                         break;
+                    case HasManyThrough::class:
```

Comment:
Let's add a note to the README documentation to explain this functionality and the limitations